op-node/rollup/derive/l1_block_info.go

```diff
@@ -187,6 +194,70 @@ func (info *L1BlockInfo) marshalBinaryInterop() ([]byte, error) {
 	return out, nil
 }
 
+// Isthmus Binary Format
+// +---------+--------------------------+
+// | Bytes   | Field                    |
+// +---------+--------------------------+
+// | 4       | Function signature       |
+// | 4       | BaseFeeScalar            |
+// | 4       | BlobBaseFeeScalar        |
+// | 8       | SequenceNumber           |
+// | 8       | Timestamp                |
+// | 8       | L1BlockNumber            |
+// | 32      | BaseFee                  |
+// | 32      | BlobBaseFee              |
+// | 32      | BlockHash                |
+// | 32      | BatcherHash              |
+// | 4       | OperatorFeeScalar        |
+// | 8       | OperatorFeeConstant      |
+// +---------+--------------------------+
+
+func (info *L1BlockInfo) marshalBinaryIsthmus() ([]byte, error) {
+	w := bytes.NewBuffer(make([]byte, 0, L1InfoIsthmusLen)) // Ecotone and Isthmus have the same length
```

Comment:
i dont fully understand the utility of this comment given the variable name

Reply:
Removed